brats/transformations.py

```diff
@@ -73,3 +76,87 @@ def hist_match(source: np.ndarray, template: np.ndarray) -> np.ndarray:
                                     template_values)
         result[positive] = interp_t_values[bin_idx]
         return result
+
+
+class NiftiOrderTransformation:
```

Comment:
the name of this guy is not very informative

Reply:
Is ```NiftiToTorchOrderTransformation``` better? Do you have any other suggestions?

Reply:
`NiftiToTorchDimensionsReorderTransformation` Very long but at least tells the truth I think xd

Reply:
Changed, I don't really have a better name.